src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
+  background-color: #fff;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
   margin: 0;
 }
+
+.header-container {
+  display: flex;
+
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.navigation {
+  display: inline-flex;
+  height: 60px;
+  align-items: center;
+  list-style-type: none;
+  gap: 20px;
+  padding: 0;
+  margin: 0;
+  line-height: 14px;
+}
+
+.list {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  padding-top: 23px;
+  padding-bottom: 23px;
+}
+
+.is-active {
+  position: relative;
+  color: rgba(0, 172, 220, 1);
```

Comment:
use some variable to store the color and reuse it here and other appropriate places